packages/assets/README.md

```diff
@@ -0,0 +1,21 @@
+# @react-native/assets-registry
+
+[![Version][version-badge]][package]
+
+## Installation
+
+```
+yarn add --dev @react-native/assets-registry
+```
+
+*Note: We're using `yarn` to install deps. Feel free to change commands to use `npm` 3+ and `npx` if you like*
+
+[version-badge]: https://img.shields.io/npm/v/@react-native/assets-registry?style=flat-square
+[package]: https://www.npmjs.com/package/@react-native/assets-registry
+
+## Testing
+
+To run the tests in this package, run the following commands from the React Native root folder:
+
+1. `yarn` to install the dependencies. You just need to run this once
+2. `yarn jest assets`.
```

Comment:
```suggestion
2. `yarn jest packages/assets`.
```

Reply:
Nice catch! Sure.
Though the current syntax works its not exactly what we would want to test as it just does regex matching.